portal-bridge/src/census.rs

```diff
@@ -161,6 +169,14 @@ impl Network {
     // peers. However, since the census continues to iterate through the peers after initialization,
     // the initialization is just to reach a critical mass of peers so that gossip can begin.
     async fn init(&mut self) {
+        let msg = match self.filter_clients.is_empty() {
+            true => format!("Initializing {} network census.", self.subnetwork),
```

Comment:
nit log line looks weird with a period IMO

```suggestion
            true => format!("Initializing {} network census", self.subnetwork),
```